src/features/references/__tests__/test-fixtures.ts

```diff
@@ -1,9 +1,10 @@
+import { UPLOADS_DIR } from '../../../io/filesystem';
 import { ReferenceItem } from '../../../types/ReferenceItem';
 
 export const REFERENCES: ReferenceItem[] = [
   {
     id: 'a_few_useful_things_to_know_about_machine_learning',
-    filepath: '/path/to/A Few Useful Things to Know about Machine Learning.pdf',
+    filepath: `/${UPLOADS_DIR}/A Few Useful Things to Know about Machine Learning.pdf`,
```

Comment:
We shouldn't need this. Do we really need this dependency on the filesystem here?